client/calypso-stubs/state/data-layer/wpcom-api-middleware.js

```diff
@@ -0,0 +1,95 @@
+/** @format */
+
```

Comment:
Since this is largely a copy-paste from https://github.com/Automattic/wp-calypso/blob/master/client/state/data-layer/wpcom-api-middleware.js, can you comment as such - pointing out the modifications? _(like we do with TaxJar)_